omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/field/PersonAttributeFragmentController.java

```diff
@@ -0,0 +1,23 @@
+package org.openmrs.module.registrationapp.fragment.controller.field;
+
+import org.openmrs.Concept;
+import org.openmrs.PersonAttributeType;
+import org.openmrs.api.context.Context;
+import org.openmrs.ui.framework.annotation.FragmentParam;
+import org.openmrs.ui.framework.fragment.FragmentModel;
+
+public class PersonAttributeFragmentController {
+
+	public void controller(FragmentModel model, @FragmentParam(required = true, value = "uuid") String uuid) throws Exception {
+		
+		PersonAttributeType personAttributeType = Context.getPersonService().getPersonAttributeTypeByUuid(uuid);
+		
+		if(personAttributeType.getFormat().contentEquals(Concept.class.getCanonicalName())){
```

Comment:
I think this should be 

`personAttributeType.getFormat().equals(Concept.class.getName())`